File: Java/Java/Factorial.java
import java.util.Scanner;

public class Factorial{
    public static void main(String args[]){
        
        int n,i,fac=1;
        Scanner s = new Scanner(System.in);
        
        System.out.print("Enter the Number : ");
        n = s.nextInt();
        
        i = 1;
        
        while(i<=n)
        {
            fac *= i;
            i++;
        }  
        
        System.out.print("Factorial is : "+fac);     
    }
}

File: C/Factorial.c
#include<stdio.h>

int main()
{
    int n,fac=1,i;
    
    printf("Enter The Number: ");
    scanf("%d",&n);
    
    for(i=1;i<=n;i++)
    {
       fac=fac*i;
    }
    
    printf("%d Factorial is : %d",n,fac);
    
    return 0;
}